### PR TITLE
ci: Fix tests that are failing with cgroupsv2

### PR DESCRIPTION
**Pull request by yaoyinnan**
Fix tests that are failing with cgroupsv2 in ubuntu 22.04.

Fixes: #4922

Signed-off-by: yaoyinnan <yaoyinnan@foxmail.com>

### CODE REVIEW

**Comment by yaoyinnan** [comment]
@liubin Please help me to approve workflows. thanks

**Comment by GabyCT** [comment]
Here are the list of tests that were skipped
Stability test with sandbox cgroup only enabled
Tracing tests
Vcpus and KSM integration tests
Nydus tests
Kata monitor tests

**Comment by yaoyinnan** [comment]
@liubin Please help me to approve workflows. thanks

**Comment by yaoyinnan** [comment]
@liubin Please help me to approve workflows. thanks

**Comment by yaoyinnan** [comment]
@liubin Please help me to approve workflows. thanks

**Comment by yaoyinnan** [comment]
@liubin Please help me to approve workflows. thanks

**Comment by yaoyinnan** [comment]
@liubin Please help me to approve workflows. thanks

**Comment by GabyCT** [comment]
@yaoyinnan it seems working properly :)

**Comment by fidencio** [comment]
@yaoyinnan, can we get this one as part of https://github.com/kata-containers/kata-containers/pull/4397 and finally have that one merged?

**Comment by yaoyinnan** [comment]
@liubin @fidencio Please help me to approve workflows `/test` and `/test-ubuntu22`. thanks

File: .ci/run.sh [hunk 1]
```diff
@@ -51,14 +51,9 @@ case "${CI_JOB}" in
...
 		echo "INFO: Skipping pmem test: Issue: https://github.com/kata-containers/tests/issues/3223"
-		if [ "${NAME}" == "Ubuntu" ] && [ "$(echo "${VERSION_ID} >= 22.04" | bc -q)" == "1" ]; then
-			issue="https://github.com/kata-containers/tests/issues/4922"
-			echo "INFO: Skipping stability test with sandbox_cgroup_only as they are not working with cgroupsv2 see $issue"
-		else
-			echo "INFO: Running stability test with sandbox_cgroup_only"
-			export TEST_SANDBOX_CGROUP_ONLY=true
-			sudo -E PATH="$PATH" CRI_RUNTIME="containerd" bash -c "make stability"
-		fi
+    echo "INFO: Running stability test with sandbox_cgroup_only"
```
Comment (liubin) [inline .ci/run.sh:54]: Please use `tab` instead of `spaces`

Comment (yaoyinnan) [reply .ci/run.sh]: Updated it

**Review by GabyCT — APPROVED** [review]

**Pull request merged**